### PR TITLE
Add menus for unstable rtcs (not used by default)

### PR DESCRIPTION
**Pull request by snozawa**
Add menus for unstable rtcs (not used by default).

### CODE REVIEW

**Comment by snozawa** [comment]
Added functions in this PR is disabled by default.
(#652 (comment))

**Comment by k-okada** [comment]
why you call
 ` execHrpsysConfiguratorCommand("hcf.startAutoBalancer()")`
instead of 
` hcf.startAutoBalancer()`

--
◉ Kei Okada

**Comment by snozawa** [comment]
Thank you for your comment.

Your comment riginally corresponds to the previous PR : https://github.com/start-jsk/rtmros_common/pull/658/.

In these PR, I used `execHrpsysConfiguratorCommand("hcf.xxx")` instead of `hcf.xxx`. 
Life cycle of dashboard is completely independent from the life cycle of rtcd. 
I'd like to enable dashboard to work even if hcf is bad condition, for example, 
- Before initializing hcf, `hcf.startAutoBalancer()` fails.
- When `hcf` has been successfully created, connected to rtcd, and after that rtcd was died and restarted, `hcf` cannot access rtcd because of CORBA error.

To enable dashbaord to work above-mentioned situation, I tried the following codes:
```
try:
   hcf.xxx # first try
except:
   setupHrpsysConfigurator() # hcf re-creation for retry
   hcf.xxx # retry
```

This code worked in that situation.

This code also required from all MenuWidegt class button press callback.
So, I tried to reduce duplicate codes by using `execHrpsysConfiguratorCommand("hcf.xxx")`.

Do you have any idea to improve that?

**Comment by k-okada** [comment]
I see, that's because of you're try to mixing ROS world and RTM world, and
my suggestion is to use everything in ROS world , to avoid issues you
mentioned. Please check how the servoOn/off are implemented, they use ROS
service call so you do not have to rtcd lifecycle.

off course I agree that sometimes we want to have  RTM only dashboard,
specially we're debugging plugins, but in that purpose we'd better to
create different dashboard and may be better to discuss in hrpsys-base, but
basically theses dashboard may used for debugging purpose for expert users
so sending command from python command line is ok.
More over these RTM dashboard is very hard to launch because you have to
 specify namesever and ports, I think most of developer is very busy on
learning how to live within ROS world and only few people (like you!) can
survive both ROS and RTM world. So tools like dashboard, which everyone
will use, need to have simple structure.


--
◉ Kei Okada
